scripts/update-se-drivers.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const cheerio = require('cheerio');
+const got = require('got');
+const chromeDriverDownloadUrl = 'https://chromedriver.chromium.org/downloads';
+const { execSync } = require('child_process');
+
+let chromeVersion;
+let chromeDriverVersion;
+let foundDriverVersion = false;
+
+function getOS() {
+  var os = process.platform;
+  if (os == "darwin") {
+    os = "MacOS";
+  } else if (os == "win32" || os == "win64") {
+    os = "Windows";
+  } else if (os == "linux") {
+    os = "Linux";
+  }
+  return os;
+}
+
+got(chromeDriverDownloadUrl).then(async(response) => {
+  os = getOS();
+  console.log(`Operating System - ${os}`);
+
+  if (os === 'MacOS') {
+    chromeVersionString = execSync(`/Applications/Google\\ Chrome.app/Contents/MacOS/Google\\ Chrome --version`) + '';
+    // Example output of chromeVersionString - Google Chrome 83.0.4103.97
+    // Split this string by spaces to get the version number
+    chromeVersion = chromeVersionString.split(" ")[2];
```

Comment:
nit:
given the version number has only digits and dot, would be easier to use regex?

Reply:
regex makes sense. Updated.